src/components/cuda/cupti_profiler.c

```diff
@@ -193,40 +193,46 @@ static int load_nvpw_sym(void)
     COMPDBG("Entering.\n");
     char dlname[] = "libnvperf_host.so";
     char *found_files[MAX_FILES];
-    char lookup_path[PATH_MAX];
-    int count, i, found = 0;
+    int count, i;
     char *papi_cuda_root = getenv("PAPI_CUDA_ROOT");
+
+    char lookup_path[PATH_MAX];
+    const char *standard_paths[] = {
+        "%s/extras/CUPTI/lib64/%s",
+        "%s/lib64/%s",
+    };
+    int num_standard_paths = sizeof(standard_paths) / sizeof(standard_paths[0]);
```

Comment:
I am not sure we should have this functionality implemented in this PR. The reason is we already have an issue tracking this problem and ideally we want a PR that addresses that issue only. This commit could be made a separate PR and merged immediately after this one. The PR can also reference the issue in the body so that merging the PR automatically closes the issue as well

Reply:
Have a look at [my fork PR](https://github.com/anustuvicl/papi/pull/2). It implements the feature in question in a commit to be merged on top of this branch.

Reply:
I will review that after we merge this